h/templates/deform/form.jinja2

```diff
@@ -0,0 +1,38 @@
+<form id="{{ field.formid }}"
+      action="{{ field.action }}"
+      method="{{ field.method }}"
+      enctype="multipart/form-data"
+      accept-charset="utf-8"
+      {% if field.css_class -%}
+      class="{{ field.css_class }}"
+      {%- endif -%}>
+  <input type="hidden" name="__formid__" value="{{ field.formid }}" />
+
+  {%- for f in field.children -%}
+    {{ field.renderer(field.widget.item_template, field=f, cstruct=cstruct.get(f.name, null)) }}
+  {% endfor -%}
+
+  <div class="form-actions">
+    <div class="form-actions-buttons">
+      {%- for button in field.buttons -%}
+        <button id="{{ field.formid + button.name }}"
+                name="{{ button.name }}"
+                type="{{ button.type }}"
+                class="btn{% if button.css_class %} {{ button.css_class }}{% endif %}"
```

Comment:
I did consider setting this up so that it would simply override it, i.e.:

```jinja2
{% if button.css_class %}{{ button.css_class }}{% else %}btn{% endif %}
```

The only reason I didn't do that is it's inconsistent with how `css_class` works everywhere else...